lua/lib/renderer.lua

```diff
@@ -48,6 +48,21 @@ if icon_state.show_file_icon then
 
 end
 
+local get_symlink_icon = function() return icon_state.icons.symlink end
+if icon_state.show_file_icon then
+  get_symlink_icon = function()
+    return #icon_state.icons.symlink > 0 and icon_state.icons.symlink.." " or ""
+  end
+end
+
+local get_special_icon = function() return icon_state.icons.default end
```

Comment:
and this too.

Reply:
Same as https://github.com/kyazdani42/nvim-tree.lua/pull/82#discussion_r466078298